### PR TITLE
Completed prep-html-syntax exercise.

### PR DESCRIPTION
**Pull request by gregg-garceau**
Untitled

### CODE REVIEW

File: prep-html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+
+<html>
+
```
Comment (thebearingedge) [inline prep-html-syntax/index.html:4]: Hi Gregg, be sure to include a `<head></head>` containing a `<title></title>` before your body too.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-html-syntax/index.html (hunk 1 above)
Comment (gregg-garceau) [reply prep-html-syntax/index.html:4]: Ah okay, will do right now

File: prep-html-syntax/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+<head>
+<title>HTML Syntax</title>
+</head>
```
Comment (gregg-garceau) [inline prep-html-syntax/index.html:7]: Untitled

File: prep-html-syntax/index.html (hunk 1 above)
Comment (gregg-garceau) [reply prep-html-syntax/index.html:4]: All set.

**Review by thebearingedge — APPROVED** [review]
Thanks!

**Pull request merged**